frontend/integration-tests/protractor.conf.ts

```diff
@@ -152,6 +152,11 @@ export const config: Config = {
       'tests/monitoring.scenario.ts',
       'tests/crd-extensions.scenario.ts',
     ]),
+    kubevirt: [
```

Comment:
Yep, makes sense.
`suite` is not used because of custom login.
@spadgett already wondered what's the reason behind the custom login scenario.
I added it because at certain point login page of openshift/console and kubevirt/web-ui were different and `login.scenario.ts` didn't work. That is not an issue now, however, `login.scenario.ts` adds numerous test cases, that are not relevant to our plugin.

Reply:
> That is not an issue now, however, `login.scenario.ts` adds numerous test cases, that are not relevant to our plugin.

OK, understood, thanks for clarification.

Reply:
> suite is not used because of custom login.

You could also modify the `suite` function to support a custom (non-standard) login scenario(s) but this can be done later on.